### ISSUE TITLE
[Documents] AsAreabrick attribute: Add custom template support

### ISSUE THREAD

**Issue by dvesh3** [pimcore/pimcore#15699]
### Improvement description

Add support for defining custom templates:

```
#[AsAreabrick(
    id: 'iframe',
    template: 'custom/location/areabricks/iframe.twig'
)]
```
discussed here https://github.com/pimcore/pimcore/pull/15406#issuecomment-1621211924

**Comment by passioneight** [comment]
@dvesh3 why not use the `#[Template('custom/location/areabricks/iframe.twig')]` that [is already available](https://symfony.com/bundles/SensioFrameworkExtraBundle/current/annotations/view.html)? 

![image](https://github.com/pimcore/pimcore/assets/57256630/7771948e-786e-48ea-8cf4-accff5538b4c)

**Comment by dvesh3** [comment]
since we are not dealing with controller here. this won't work for areabricks, right?

**Comment by passioneight** [comment]
Yes, we'd have to support this explicitly. But the same holds true for the `template` parameter suggested for the `AsAreabrick` attribute.

So, I imagine that using the `Template` attribute comes with the advantage of auto-completion in IDEs, as well as the possibility to support other functionality in the future (if needed) because the `Template` attribute allows for more than just configuring the template.

So, maybe the support for this attribute is still worth it. Wdyt?

**Comment by jdreesen** [comment]
> @dvesh3 why not use the `#[Template('custom/location/areabricks/iframe.twig')]` that [is already available](https://symfony.com/bundles/SensioFrameworkExtraBundle/current/annotations/view.html)?
> 
> ![image](https://user-images.githubusercontent.com/57256630/259408850-7771948e-786e-48ea-8cf4-accff5538b4c.png)

I think we should use Symfony's native [Symfony\Bridge\Twig\Attribute\Template](https://symfony.com/doc/current/templates.html#templates-template-attribute) in that case, instead of the one from the `SensioFrameworkExtraBundle`, which is deprecated.
